### PR TITLE
[new-app] Settings Page

### PR DESCRIPTION
**Pull request by jbaek7023**
working on it! :)

### CODE REVIEW

File: src/sass/_common/base/constants.scss [hunk 1]
```diff
@@ -36,6 +36,7 @@ $COLOR_LIGHT_GRAY    : $fillColor;     // #f2f2f2
 $COLOR_GRAY          : $borderColor;   // #dedede
 $COLOR_DARK_GRAY     : #44484c;
 $COLOR_GREEN         : $positiveColor; // #2e8836
+$COLOR_LETTER_GRAY   : #333;
```
Comment (binary-hamedanchi) [inline src/sass/_common/base/constants.scss]: Probably meant to put in `src/sass/app_2/_common/base/constants.scss` as it's going to be used in app_2 only.

Comment (jbaek7023) [reply src/sass/_common/base/constants.scss]: got it thanks :)

File: src/javascript/app_2/pages/statement/statement.jsx [hunk 2]
```diff
@@ -167,85 +108,17 @@ class Statement extends React.PureComponent {
...
-        );
     }
 
     renderNoActivityMessage() {
         return (
             <div className='container'>
                 <div className='statement__no-activity-msg'>
                     {
-                        !this.state.date_from && !this.state.date_to
+                        !this.props.date_from && !this.props.date_to
```
Comment (jbaek7023) [inline src/javascript/app_2/pages/statement/statement.jsx]: Code can be more readable when you define props explicitely.

1) You can define props 
`const { date_from, date_to } = this.props;`

2) use the props name directly
`date_from && date_to ? local blah blah`

File: src/javascript/app_2/pages/statement/statement.jsx [hunk 3]
```diff
@@ -270,9 +143,9 @@ class Statement extends React.PureComponent {
                         name='date_from'
                         initial_value=''
                         placeholder={localize('Start date')}
-                        startDate={this.state.date_to || today}
-                        maxDate={this.state.date_to || today}
-                        onChange={this.handleDateChange}
+                        startDate={this.props.date_to || today}
```
Comment (jbaek7023) [inline src/javascript/app_2/pages/statement/statement.jsx]: same goes here :)

File: src/javascript/app_2/pages/statement/statement.jsx [hunk 4]
```diff
@@ -2,44 +2,16 @@ import React              from 'react';
 import moment             from 'moment';
```
Comment (jbaek7023) [inline src/javascript/app_2/pages/statement/statement.jsx]: This refactoring doesn't really relate to this PR, "Settings page". 

Maybe can you create new PR for `Make Stateless component with MobX model/store`?

**Pull request closed without merging**